uuid.go

```diff
@@ -210,6 +210,20 @@ func (u UUID) String() string {
 	return string(buf)
 }
 
+// Returns string representation of UUID without dashes:
+// xxxxxxxx(-)xxxx(-)xxxx(-)xxxx(-)xxxxxxxxxxxx.
+func (u UUID) StringWithoutDashes() string {
+	buf := make([]byte, 32)
+
+	hex.Encode(buf[0:8], u[0:4])
+	hex.Encode(buf[8:12], u[4:6])
+	hex.Encode(buf[12:16], u[6:8])
+	hex.Encode(buf[16:20], u[8:10])
+	hex.Encode(buf[20:], u[10:])
+
+	return string(buf)
```

Comment:
is it better to use `hex.EncodeToString(u.Bytes())`?

Reply:
Yes, it is better and more coincise. Thank you for the advice. Tomorrow I will push it.